frontend/src/lib/email/Email.svelte

```diff
@@ -0,0 +1,23 @@
+﻿<script lang="ts">
+    import Subject from "$lib/email/Subject.svelte";
+
+    const silLogo = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkAgMAAAANjH3HAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAMUExURSh2xABcuVaT0fj7/aZ5vGgAAAAJcEhZcwAALiAAAC4gAdUcHhsAAAEQSURBVEjH7dS7DcIwEAZglsgSNIzACEGEMygUGYEpaOjdU4I3oKLKCIxAQZkuSODDzgOi5P6CQJlrIvmTT+c7xyNCMcggg/wsC7MHMmG+EY2NptgcmzJnZkt05pRCfjQlhDJFsnDgF7qy9PKUJIKy9nKXJGHOZydJXNHaf7uyYSYk9o/iKghkcVVfZHEnzWpJ2t2xsijXnFTMRmXb0OQCKVsx7VTc4weUieLW5MkV6eS7U9QdiOLTaVlC3zpR1r5sUdyVu34rqseeCErYrc2UwZ/z2HJlxO+oe1BFQwiJhXKHcoWi6xvSlpyA2C21s1VR/Avlu1PF8F73kSMSZZDERgNZHZDEOyRq931t1OekRC8admBBuTG0EgAAAABJRU5ErkJggg==";
+    export let subject: string;
```

Comment:
it an be helpful to see exports right after imports

Reply:
sorry your message got a little garbled. You want me to move the export above the `silLogo` variable?

Reply:
sorry about that.  Yes, it's typically better to have your component api immediately visible to readers, so that would mean `export` would typically be the next thing you see after your `import`'s